pkg/proxy/iptables/proxier_test.go

```diff
@@ -1268,11 +1268,8 @@ func Test_accumulateEndpointsMap(t *testing.T) {
 				&endpointsInfo{"1.1.1.1:11", false},
 			},
 		},
-		expectedNew: map[proxy.ServicePortName][]*endpointsInfo{},
-		expectedStale: []endpointServicePair{{
-			endpoint:        "1.1.1.1:11",
-			servicePortName: makeServicePortName("ns1", "ep1", ""),
-		}},
+		expectedNew:   map[proxy.ServicePortName][]*endpointsInfo{},
+		expectedStale: []endpointServicePair{},
```

Comment:
But you *did* change behavior; you had to change the expected results in proxier_test.go to match the behavior change in accumulateEndpointsMap(). And there are now no test cases with a non-empty expectedStale, so whatever it is that staleConnections is supposed to represent at this point (which I can't even figure out), you're not testing it.